docs/release-notes.rst

```diff
@@ -10,6 +10,26 @@
  Version 0.28
 **************
 
+Version 0.28.1
+==============
+
+**Release Date:** February 20, 2024
+
+**Improvements**
+
+-  GCS Storage client will now retry following the default policy on ``TooManyRequests`` rate limit
+   errors.
+
+**Bug Fixes**
+
+-  Since 0.26.2, it was possible to cause Determined Trials and Commands to hang after the main
```

Comment:
```suggestion
-  Since 0.26.2, it was possible to cause Determined trials and commands to hang after the main
```

Reply:
updated the usage guide https://hpe-aiatscale.atlassian.net/wiki/spaces/DOC/pages/1332707345/Usage+Guide